Gems/PhysX/Core/Code/Source/ArticulationLinkComponent.cpp

```diff
@@ -223,6 +223,20 @@ namespace PhysX
     void ArticulationLinkComponent::OnTransformChanged(
         [[maybe_unused]] const AZ::Transform& local, [[maybe_unused]] const AZ::Transform& world)
     {
+#if (PX_PHYSICS_VERSION_MAJOR == 5)
+        if (m_enabled)
+        {
+            return;
+        }
+        AZ_Warning("ArticulationLinkComponent", IsRootArticulation(), "Pose can be adjusted only for the root articulation link.");
+        if (m_articulation && IsRootArticulation())
+        {
+            physx::PxArticulationKinematicFlags kinematicFlag {};
+            kinematicFlag.raise(physx::PxArticulationKinematicFlag::ePOSITION);
+            m_articulation->setRootGlobalPose(PxMathConvert(world));
+            m_articulation->updateKinematic(kinematicFlag);
```

Comment:
I take it these APIs are only available in physx5 with articulation?

Reply:
Yes it is only PhysX5